cors/cors-scala/LICENSE

```diff
@@ -0,0 +1,15 @@
+This software is licensed under the Apache 2 license, quoted below.
+
+Copyright 2016 Lightbend Inc. [http://www.lightbend.com]
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License. You may obtain a copy of
+the License at
+
+    [http://www.apache.org/licenses/LICENSE-2.0]
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations under
+the License.
```

Comment:
Akka makes all of its examples public domain. I wonder if we should do the same.

https://github.com/akka/akka-samples/blob/2.5/akka-sample-camel-java/LICENSE

I'll clarify with Derek.

Reply:
TIL

I wasn't sure so I preferred staying on the safe side. Thanks for checking this. Should we remove all the `LICENSE`s on a separate PR?

Reply:
Yeah, no need to block this.

Reply:
Confirming that we will keep these recipes under the Apache 2 license.

Reply:
As an improvement, may we move the license to the top of the repo and forget about it on _each_ recipe?

Reply:
I don't see why not